### PR TITLE
startVideoPreviewForVideoInput uses the active video input stream instead of calling getUserMedia

### PR DESCRIPTION
**Pull request by ltrung**
**Description of changes:**
Currently `startVideoPreviewForVideoInput` will call `getUserMedia` to re-select the active chosen video device input.
This isn't necessary as we already call `getUserMedia` in `chooseVideoInputDevice` which is required before calling `startVideoPreviewForVideoInput`.
This also causes synthesized video input stream (such as SMTP bar) to not working for video preview because `getUserMedia` will just replace it with another video stream.

**Testing**

1. Have you successfully run `npm run build:release` locally? Yes
2. How did you test these changes? In Video Preview screen in the meeting demo, switch to `Blue` or `SMTP` options and verify that the video preview shows up correctly.
3. Can these changes be tested using the demo application? If yes, which demo application can be used to test it? 
Yes, meeting demo.
4. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved? N/A
5. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved? N/A

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by richnew10** [comment]
@ltrung could you rebase this? It looks like it has some churn.

**Comment by ltrung** [comment]
> @ltrung could you rebase this? It looks like it has some churn.

Could you clarify? I double check the committed files and they look correct to me.

**Comment by richnew10** [comment]
> > @ltrung could you rebase this? It looks like it has some churn.
> 
> Could you clarify? I double check the committed files and they look correct to me.

nm — Git was showing a stale diff for some reason.

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 ### Changed
+- `startVideoPreviewForVideoInput` uses the active video input stream instead of calling `GetUserMedia` again
```
Comment (richnew10) [inline CHANGELOG.md:13]: Nit: `getUserMedia`.

File: test/devicecontroller/DefaultDeviceController.test.ts [hunk 2]
```diff
@@ -2257,21 +2257,10 @@ describe('DefaultDeviceController', () => {
...
-      spy.restore();
-    });
-
-    it('does not disconnect or connect the video stream for preview if no active video exsits', async () => {
+    it('does not disconnect or connect the video stream for preview if no active video exists', () => {
       const spy1 = sinon.spy(DefaultVideoTile, 'disconnectVideoStreamFromVideoElement');
       const spy2 = sinon.spy(DefaultVideoTile, 'connectVideoStreamToVideoElement');
       deviceController.startVideoPreviewForVideoInput(element);
-      await delay(100);
       expect(spy1.called).to.be.false;
```
Comment (richnew10) [inline test/devicecontroller/DefaultDeviceController.test.ts:2264]: Can we get a test to prove that it only calls `gUM` once?

**Review by richnew10 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (ltrung) [reply CHANGELOG.md]: Fixed.

File: test/devicecontroller/DefaultDeviceController.test.ts (hunk 2 above)
Comment (ltrung) [reply test/devicecontroller/DefaultDeviceController.test.ts:2264]: We remove the getUserMedia call so there would be no call right?

Comment (richnew10) [reply test/devicecontroller/DefaultDeviceController.test.ts:2264]: There should be a single call when they choose the video input, and no additional call when they start the preview.

This is easy to do with sinon.spy; see elsewhere in the codebase.

Comment (simmkyu) [reply test/devicecontroller/DefaultDeviceController.test.ts:2264]: Not in this `it` block, you can validate that in the previous `it`. In this `it`, you can ensure that `gUM` is not called.

Comment (ltrung) [reply test/devicecontroller/DefaultDeviceController.test.ts]: Thanks. Updated in both test case to check for getUserMedia.

**Review by simmkyu — APPROVED** [review]

**Review by simmkyu — APPROVED** [review]

**Review by richnew10 — APPROVED** [review]

**Pull request merged**